ocs_ci/ocs/utils.py

```diff
@@ -1284,3 +1284,20 @@ def thread_init_class(class_init_operations, shutdown):
         if shutdown == 0:
             executor.shutdown(wait=True)
             return
+
+
+def get_catalogsource(name):
```

Comment:
please check if we can use functionalities from ocs_ci.ocs.resources.catalog_source.CatalogSource

Reply:
I need to drop that function, I'm initiating LVM and he does also check catalogsource for LVM version so I'll drop it.